assets/app/scripts/services/routes.js

```diff
@@ -62,6 +62,21 @@ angular.module("openshiftConsole")
       }
     };
 
+    var addIngressWarnings = function(route, warnings) {
+      angular.forEach(route.status.ingress, function(ingress) {
+        angular.forEach(ingress.conditions, function(condition) {
+          if (condition.type === "Admitted" && condition.status === "False") {
```

Comment:
Actually going to do
```
        var condition = _.find(ingress.conditions, { type: "Admitted", status: "False" });
        if (condition) {
```
because i need access to the condition to get the reason/message